.github/workflows/test-node.yml

```diff
@@ -47,14 +49,17 @@ jobs:
 
       - run: npm run build:esm
 
+      - name: Create reports directory
```

Comment:
why do we create and save reports? Is it to be displayed in the pipeline later? 
I know before we were doing so to push allure reports but why this one is done?

Reply:
similar situation, we generate the reports and use GH UI to display it